DemoViper/Sources/Services/Network/Network.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Network.swift
+//  DemoViper
+//
+//  Created by Su Van Ho on 13/8/19.
+//  Copyright © 2019 Nimble Co. Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class Network {
+    func repoNetwork() -> RepoNetwork {
```

Comment:
I think `Network` can be omitted, this is kind of duplicated with class name 🤔

```swift
var repository: RepoNetwork { return .default } 

var image: ImageNetwork { return .default }
```

Reply:
👍